base/linalg/cholesky.jl

```diff
@@ -101,16 +101,14 @@ function _chol!(A::AbstractMatrix, ::Type{LowerTriangular})
             end
             A[k,k] = Akk
             AkkInv = inv(Akk)
-            for j = 1:k
-                for i = k + 1:n
-                    if j == 1
-                        A[i,k] = A[i,k]*AkkInv'
-                    end
-                    if j < k
-                        A[i,k] -= A[i,j]*A[k,j]'*AkkInv'
-                    end
+            for j = 1:k - 1
+                @simd for i = k + 1:n
```

Comment:
kind of misleading precedence vs spacing here - was like that way before, but now there are 3 instances of it instead of 1